genvm/server.py

```diff
@@ -76,11 +80,17 @@ def validator_executes_transaction(
 ) -> dict:
 
     delete_recipts()
+    
+    msg = MessageHandler(app, socketio)
 
     return_data = {"status": "error", "data": None}
 
     icontract_file, recipt_file, _, _ = transaction_files()
 
+    msg.debug_response("icontract", icontract)
```

Comment:
```suggestion
    msg.debug_response("Contract Code", contract_code)
```

Reply:
There is no variable called `contract_code`.